### PR TITLE
Tour Info Tooltip and fixes

### PR DESCRIPTION
**Pull request by FJBDev**
- Fix for the Fit Export speed value

Tour Info Tooltip (Frederic)
  - Fixed a bug where the time formats were not correctly applied
  - Added, when available, the time and percentage of time spent in each HR zone

image

- Fix for the weather retrieval logging when only the air quality is retrieved (https://github.com/mytourbook/mytourbook/issues/1076)

image

- Added an icon to each cloud uploader option

image

- Added a Nebula switch button to show/hide the UI secrets

image
image

### CODE REVIEW

**Review by wolfgang-ch — APPROVED** [review]
2 small issues

**Review by wolfgang-ch — COMMENTED** [review]
One more issue, your PR was not merged with the latest main branch

image

**Comment by wolfgang-ch** [comment]
2 other issues

* HR zone values are propably clipped
* The HR zone sorting should be the same as in the `Training` view

image

**Comment by FJBDev** [comment]
> image

Is that a single tour or multiple tour ?

**Comment by FJBDev** [comment]
> One more issue, your PR was not merged with the latest main branch
> 

Ok, I forgot to see that you had recent changes. Did  that cause any unwanted overwritten commits ?

File: bundles/net.tourbook.common/src/net/tourbook/common/time/TimeTools.java [hunk 1]
```diff
@@ -293,6 +293,8 @@ public class TimeTools {
       };
```
[the remark this answers is not in the document]
Comment (FJBDev) [reply bundles/net.tourbook.common/src/net/tourbook/common/time/TimeTools.java:293]: Ok, I will fix that

File: bundles/net.tourbook.common/src/net/tourbook/common/UI.java [hunk 2]
```diff
@@ -1201,6 +1201,17 @@ public static float convertPressure_ToMetric(final float weatherPressure) {
       return weatherPressure / 0.02953f;
    }
 
+   /**
+    * Convert a speed value from km/h to m/s
+    *
+    * @param speed
+    * @return Returns the speed value in m/s.
+    */
+   public static float convertSpeed_FromKphToMps(final float speed) {
```
[the remark this answers is not in the document]
Comment (FJBDev) [reply bundles/net.tourbook.common/src/net/tourbook/common/UI.java:1210]: I will rename this function

**Comment by wolfgang-ch** [comment]
> > One more issue, your PR was not merged with the latest main branch
> 
> Ok, I forgot to see that you had recent changes. Did that cause any unwanted overwritten commits ?

No, this commit contained only one file change

**Comment by wolfgang-ch** [comment]
> Is that a single tour or multiple tour ?

It's a single tour

After analyzing the code, the displayed value is OK for the related **recording time** but the `Training` view is related to the **moving time**

File: bundles/net.tourbook.common/src/net/tourbook/common/time/TimeTools.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (FJBDev) [reply bundles/net.tourbook.common/src/net/tourbook/common/time/TimeTools.java:293]: Fixed in [this PR](https://github.com/mytourbook/mytourbook/pull/1087)

File: bundles/net.tourbook.common/src/net/tourbook/common/UI.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (FJBDev) [reply bundles/net.tourbook.common/src/net/tourbook/common/UI.java:1210]: Fixed in [this PR](https://github.com/mytourbook/mytourbook/pull/1087)

**Comment by FJBDev** [comment]
>     * The HR zone sorting should be the same as in the `Training` view
> 
> 

Fixed in [this PR](https://github.com/mytourbook/mytourbook/pull/1087)

**Comment by wolfgang-ch** [comment]
> > Is that a single tour or multiple tour ?
> 
> It's a single tour
> 
> After analyzing the code, the displayed value is OK for the related **recording time** but the `Training` view is related to the **moving time**

I adjusted the related time by using the moving time in https://github.com/mytourbook/mytourbook/commit/531761c74f7d3fc19710ec4f8341768cdb921b6b

**Pull request merged**